native/NativeSampleApps/MobileSyncExplorer/src/com/salesforce/samples/mobilesyncexplorer/ui/DeleteDialogFragment.kt

```diff
@@ -24,46 +24,29 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-package com.salesforce.samples.mobilesyncexplorer.ui;
+package com.salesforce.samples.mobilesyncexplorer.ui
 
-import android.app.AlertDialog;
-import android.app.Dialog;
-import android.app.DialogFragment;
-import android.content.DialogInterface;
-import android.os.Bundle;
-
-import com.salesforce.samples.mobilesyncexplorer.R;
+import android.app.AlertDialog.Builder
+import android.app.Dialog
+import android.content.DialogInterface
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import com.salesforce.samples.mobilesyncexplorer.R.string.cancel
+import com.salesforce.samples.mobilesyncexplorer.R.string.delete_title
+import com.salesforce.samples.mobilesyncexplorer.R.string.yes
 
 /**
  * A simple dialog fragment to provide options for deletion.
- *
- * @deprecated The exact signature of the methods inside of DeleteDialogFragment and its inherited
- * methods may change in Mobile SDK 11.0 when the deprecated base class {@link android.app.DialogFragment} is
- * replaced with {@link androidx.fragment.app.DialogFragment}.
  */
-@Deprecated
-public class DeleteDialogFragment extends DialogFragment {
-
-	/**
-	 * Default constructor.
-	 */
-	public DeleteDialogFragment() {
-	}
-
-	@Override
-	public Dialog onCreateDialog(Bundle savedInstanceState) {
-		return new AlertDialog.Builder(getActivity())
-				.setTitle(R.string.delete_title)
-				.setPositiveButton(R.string.yes,
-				new DialogInterface.OnClickListener() {
+class DeleteDialogFragment : DialogFragment() {
```

Comment:
One of two crux changes is migrating the dialogs plus a Kotlin conversion.  Logically, this is pretty much the same.